.devcontainer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:lts
+
+WORKDIR /app
+
+# Set an environment variable to be able to detect we are in dev container
+ENV NODE_ENV=devcontainer
+
+EXPOSE 3000
+COPY ./ /app
```

Comment:
You should not copy in the source...
We don't want to work with a copy, we want to work with the actual thing (which is automatically mounted by vscode)

Reply:
Cheers I was working of an old Dockerfile from Docusaurus v1